strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.yaml

```diff
@@ -318,6 +320,10 @@ configuration:
               type: npm
               lastChangeId: 0
               replicateUrl: https://replicate.npmjs.com/
+          repositoryConfiguration:
+            type: npm
+            cronEnabled: true
+            cronExpression: "0 0 * ? * * *"
```

Comment:
I'm not sure I get your idea about this. A repository can have multiple cron tasks associated with it. Each of these cron tasks might have to run at a different time. If you run all the cron tasks for the repository at the same time, then you might hit performance issues.